mock-caas/src/main/java/com/sequenceiq/caas/grpc/service/MockUserManagementService.java

```diff
@@ -506,6 +507,34 @@ public void listRoles(UserManagementProto.ListRolesRequest request, StreamObserv
         responseObserver.onCompleted();
     }
 
+    @Override
+    public void getActorWorkloadCredentials(com.cloudera.thunderhead.service.usermanagement.UserManagementProto.GetActorWorkloadCredentialsRequest request,
```

Comment:
Does the response here actually allow users to log in? What's the password used to generate these values? We need to have some actual values here otherwise folks running locally won't be able to log into their clusters.